constellation-app/constellation
# Fix Build graph from image

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change
- Modified to add only one file filter for multiple extensions in the FileInputPane
- Allow multiple images to be selected (and validate semicolon delimited string of paths)
- Placement of multiple images on the graph


### Alternate Designs



### Why Should This Be In Core?
Otherwise Build Graph -> From Image cannot filter/load multiple images of image types properly.



### Benefits
To be able to build a graph by loading multiple images files.


### Possible Drawbacks
Not able to build a graph by loading multiple images files.


### Verification Process
1. Add new graph.
2. Select Experimental -> Build Graph -> From Image...
3. Select multiple images.

Verify that images appear on the graph with no exceptions or warnings.


### Applicable Issues
#2146
